src/Collection/CollectionTrait.php

```diff
@@ -90,8 +90,12 @@ public function filter(?callable $callback = null): CollectionInterface
     /**
      * @inheritDoc
      */
-    public function reject(callable $callback): CollectionInterface
+    public function reject(?callable $callback = null): CollectionInterface
     {
+        $callback ??= function ($v) {
+            return (bool)$v;
```

Comment:
Maybe it should default to behave the same as default `filter()` and reject falsey values.

Reply:
Hrm. Having it be the inverse of filter() is kind of useful. It gives you a simple way to get all the falsey values :shrug: A similar implementation can be found in [elixir's enum package](https://hexdocs.pm/elixir/1.12/Enum.html#reject/2) as does [ruby's stdlib](https://apidock.com/ruby/v2_5_5/Array/reject).

Reply:
The default should be the inverse of filter().